packages/authentication/src/web.ts

```diff
@@ -278,6 +445,10 @@ export class FirebaseAuthenticationWeb
     await auth.signOut();
   }
 
+  public async unlink(): Promise<void> {
+    throw new Error('Not available on web.');
```

Comment:
We should implement this (not sure if this is what you had in mind).
In addition, you have to pass the provider that should be unlinked.
Strictly taken, we would have to follow the naming `unlinkFromGoogle`, `unlinkFromApple`, etc. again here.
But probably that would be a bit much. Otherwise start with one parameter and pass the ProviderId/SignInMethod and we'll see how it looks at the end.
I had not even thought of this method.

Reply:
> We should implement this (not sure if this is what you had in mind). In addition, you have to pass the provider that should be unlinked. Strictly taken, we would have to follow the naming `unlinkFromGoogle`, `unlinkFromApple`, etc. again here. But probably that would be a bit much. Otherwise start with one parameter and pass the ProviderId/SignInMethod and we'll see how it looks at the end. I had not even thought of this method.

Yes, you are right. But the `unlink` method is simply using the `providerId` as a parameter which makes it easy to use. I therefore made the `ProviderId` const exportable, so that developers can use this as a reference.